var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -41,12 +41,16 @@ class Pfunit(CMakePackage):
     variant('shared', default=True,
             description='Build shared library in addition to static')
     variant('mpi', default=False, description='Enable MPI')
+    variant('use-comm-world', default=False, description='Enable MPI_COMM_WORLD for testing')
```

Comment:
I prefer variants to use underscores over dashes as the minus sign is used to disable variants, and `+use-comm-world` can seem ambiguous for new users.

Reply:
Fixed!